spl/src/associated_token.rs

```diff
@@ -4,7 +4,9 @@ use anchor_lang::Result;
 use anchor_lang::{context::CpiContext, Accounts};
 
 pub use spl_associated_token_account::{
-    get_associated_token_address, get_associated_token_address_with_program_id, ID,
+    get_associated_token_address, get_associated_token_address_with_program_id,
+    instruction::{create_associated_token_account, create_associated_token_account_idempotent},
+    ID,
```

Comment:
Not a big difference, but we already export other crates e.g. `spl_token`:

https://github.com/coral-xyz/anchor/blob/efe2e40c12af2f17f38637350fc3a214f746aefe/spl/src/token.rs#L8

So to make it consistent, what do you think of exporting `spl_associated_token_account` here?